packages/gotrue/lib/src/gotrue_client.dart

```diff
@@ -630,6 +624,8 @@ class GoTrueClient {
   }) async {
     if (_flowType == AuthFlowType.pkce) {
       final authCode = originUrl.queryParameters['code'];
+      final redirectType = originUrl.queryParameters['type'];
```

Comment:
Actually, the `type` query parameter does not exist, and the only parameter we have is `code`. 

I think fixing this issue would require saving additional data in the local storage. Let's wait to see how js lib fixes it and match what they do. 
https://github.com/supabase/gotrue-js/pull/813